includes/html/api_functions.inc.php

```diff
@@ -545,7 +547,7 @@ function maintenance_device(Illuminate\Http\Request $request)
         $device->save();
     }
 
-    if ($request->json('start')) {
+    if ($data['start']) {
```

Comment:
This needs an isset()
```suggestion
    if (isset($data['start'])) {
```